routes/web.php

```diff
@@ -60,23 +60,15 @@
 /*
 *   Controller Dashboard
 */
-Route::prefix('dashboard')->middleware('auth')->group(function() {
+Route::prefix('dashboard')->middleware('auth')->group(function () {
     Route::get('/', 'ControllerDash@dash');
 
-    Route::prefix('controllers')->group(function() {
-      
+    Route::prefix('controllers')->group(function () {
         Route::get('/mentoravi', 'TrainingController@showMentAvail');
         Route::post('/mentoravi', 'TrainingController@saveSession');
         Route::get('/trainingreq', 'TrainingController@showRequests');
         Route::delete('/trainingreq/{id}/delete', 'TrainingController@cancelSession');
         Route::get('/sessions_today', 'TrainingController@showsessions_today');
-
```

Comment:
Thanks for fixing this whitespace... I don't know why that was in the routes file